docs/_articles/en/apex/writing.md

```diff
@@ -61,7 +65,7 @@ Apex extension provides go to definition support for user-defined Apex such as c
 
 ## Find All References
 
-You can find all references to user-defined Apex such as Classes, Class variables, Enums, Interfaces, Methods, and Properties. To find references, select an item and press Shift+F12; Or right-click the item and select **Find All References**. The reference results are displayed in the left pane of the editor window.
+You can find all references to user-defined Apex such as Classes, Class variables, Enums, Interfaces, Methods, and Properties. To find references, right-click the item and select **Go To References** or press Shift+F12; Or right-click the item and select **Find All References**. The reference results are displayed in the left pane of the editor window.
```

Comment:
once again, not sure if Classes, Enums, and such need to be capitalized. Are they in the Apex docs?